kernel/src/ipc.rs

```diff
@@ -139,10 +153,10 @@ impl<const NUM_PROCS: usize> LegacyDriver for IPC<NUM_PROCS> {
             0 => self
```

Comment:
0 has to be the "exists" command; you should bump all the command IDs by 1:

https://github.com/tock/tock/blob/tock-2.0-dev/doc/reference/trd-syscalls.md#431-command-identifier-0